text/0000-pack-buildpack-create.md

```diff
@@ -0,0 +1,80 @@
+# Meta
+[meta]: #meta
+- Name: Pack Command to Create a Buildpack Repo
+- Start Date: 2021-01-19
+- Author(s): [jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new command in Pack that would create the scaffolding for a new buildpack.
+
+# Definitions
+[definitions]: #definitions
+
+- *Scaffold* - the essential files and directory required for a buildpack
+
+# Motivation
+[motivation]: #motivation
+
+Every new buildpack requires a few essential files and directories. We can make buildpack development easier by automatically generating these with a single command.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Add a new command to Pack:
+
+```
+$ pack buildpack create <id> \
+    --language <lang> \
+    --stack <stack>
+```
+
+Where the flags include:
+
+* `language` - a choice of programming languages to use for the template. Pack will support Bash and Go at a minimum.
```

Comment:
How does one get a language in this list? Will we support 3rd party at all?

Reply:
+1 on defaulting to `bash`, even when `pack` runs on Windows. 

Most Windows desktop users today would likely be building Linux buildpacks (on LCOW daemon). If they want to create a buildpack for WCOW, right now, there's not a clear user-story for the rest of the experience given there's currently not a beginner-friendly ecosystem of Windows stacks/builders anyways. They would be likely be better served by a separate samples/template/tutorial walk-through for the time being.

Reply:
Thanks @micahyoung!

We also discussed in the working group that in the future we may add a `--template` option, and curate a repository of templates for different languages. Contributors could PR into the repo to add support for other langs (or even for things like packit).

Reply:
This is very interesting! An alternative to consider would be having a functionality similar to [cookiecutter](https://github.com/cookiecutter/cookiecutter) where you can provide a URI to a template (either a VCS uri or on the local fs). This would make it easier to handle new template contributions and they don't need to be put in a single repository.